### ISSUE TITLE
Add accessibility statement

### ISSUE THREAD

**Issue by jaronheard** [hackoregon/civic#1187]
**Is your feature request related to a problem? Please describe.*
Currently, a user of our site can't find our approach to accessibility, and there is no way to contact us about accessibility issues.

**Describe the solution you'd like**
An accessibility statement should be added, incorporating the following components:
• Add a page to the website at civicplatform.org/accessibility
• On that page, describe our approach to accessibility. This can excerpt from the content here: https://hackoregon.github.io/civic/?path=/story/design-ux-style-guide--accessibility
• Add a contact email (feedback-grp@civicsoftwarefoundation.org) so that reported issues can be added as GitHub issues.
• Add a link to the accessibility statement to the footer of the homepage

**Describe alternatives you've considered**
• Could link directly to https://hackoregon.github.io/civic/iframe.html?id=design-ux-style-guide--accessibility

**Additional context**
Here's some information about accessibility statements from https://www.youtube.com/watch?v=uNuCy9wkszM

MARCY SUTTON: Accessibility statements are great tools no matter what kind of a website you're making, whether it's with heavy JavaScript or not. So an accessibility statement is generally a page on your website that's easy to find, maybe it's linked your website footer, and it has things like what you're doing to improve accessibility, maybe what level of the Web Content Accessibility Guidelines that you're aiming for. It's nice to have that goal and that target whether or not you've actually met it, but you have to keep actively working at that to improve. You can also collect any accessibility tips or information about keyboard shortcuts or ways to use your website for accessibility, and ways for users to contact you. That's one of the most important pieces, having an affirmative statement that says, hey, we might not be perfect at this, but we'd love your feedback and get in touch with us. And if people do, act on that feedback. So it's opening that conversation to bring people in and make them feel included and give them a way to give you feedback. Because a lot of these websites that have glaring accessibility issues, we have no way to contact them. So you might see some tweets of people calling out companies because they can't use the website or the service. Maybe an update to the website or application breaks what used to work. So if you have that statement, it gives people a way to contact you in an official channel so that you can act on that feedback.
